_sass/_card.scss

```diff
@@ -58,7 +78,7 @@
 
     @media only screen and
     (max-width: $screen-sm) {
-      font-size: 16px;
+      font-size: 22px;
```

Comment:
This isn't scoped specifically to the blog, but I think it looks pretty good. It affects the card-titles in the cards on the landing page and the about page. It's specific for max-width ~768px (double check max-width value in the _variables.scss file)

Leaving this in for now.